### PR TITLE
[WFLY-17871] added test-case for managed executor service defined via…

### PR DESCRIPTION
**Pull request by istudens**
… annotation

This is a test-case for managed executor service defined via annotations. I can't see any such test in the test-suite.

Issue: https://issues.redhat.com/browse/WFLY-17871

### CODE REVIEW

**Comment by kabir** [comment]
@istudens I see a failure in the new test case https://ci.wildfly.org/buildConfiguration/WF_PullRequest_GalleonLinuxJdk11/366184?expandBuildProblemsSection=true&hideProblemsFromDependencies=false&expandBuildTestsSection=true&hideTestsFromDependencies=false&expandBuildChangesSection=true

**Comment by darranl** [comment]
@emmartins it looks like this one is waiting for a review from you?

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationTestCase.java
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationTestCase.java:21]: Please update to the current copyright/license header.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java:21]: Please update to the current copyright/license header.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/ManagedExecutorServiceAnnotationBean.java:36]: import static should come first, then import java, then the rest.

**Review by bstansberry — CHANGES_REQUESTED** [review]
@istudens Formatting stuff aside, this is fine. If you see this and update the formatting stuff, please ping me in zulip so I know to check on it.

**Comment by istudens** [comment]
@bstansberry done

**Review by bstansberry — APPROVED** [review]

**Review by emmartins — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @istudens

**Pull request merged**
